meta-oe/recipes-support/avro/avro-c++/0001-Remove-cpp-unittest-compilation.patch

```diff
@@ -1,9 +1,8 @@
 From 6be1c45e1f7ee78499bdca76c25ec23a61317134 Mon Sep 17 00:00:00 2001
 From: Rui Costa <rui.ms.costa@ctw.bmwgroup.com>
 Date: Wed, 20 Mar 2024 14:58:27 +0000
-Subject: [PATCH 1/2] Remove cpp unittest compilation
+Subject: [PATCH 1/3] Remove cpp unittest compilation
 
-Upstream-Status: Inappropriate [oe-specific]
```

Comment:
Don't drop Upstream-Status and better to use --no-numbered and --no-signature with git format-patch to avoid these unnecessary changes in .patch files (like devtool does).

Reply:
Right I forgot when I updated. Done